workshops.md

```diff
@@ -1,23 +1,23 @@
 ---
 title: "Phase Field Workshops"
 layout: workshop
-comment: Workshops at Northwestern to develop benchmark problems
+comment: Workshops at Northwestern University to develop benchmark problems
 collections:
   - header: Workshops
     tag: workshops
     js: workshops
     comment:
 ---
 
-The workshops and associated [hackathons]({{ site.baseurl
-}}/hackathons) are integral to the development effort of the [phase
-field benchmark problems]({{ site.baseurl }}/#benchmarks). The
-workshops bring together a spectrum of researchers from across the
-United States and internationally, including both graduate students
-and senior researchers. They help us formulate, test, and revise the
-benchmark problems and provide other feedback, such as information for
-developers of open-source codes on how easily their codes can be
-installed. Significant portions of the workshop are devoted to
+An integral component of the CHiMaD [phase field benchmarking effort]({{ site.baseurl }}/#benchmarks)
+is regular workshops and associated [hackathons]({{ site.baseurl
+}}/hackathons).  Approximately twice a year, a spectrum of researchers
+from across the United States and internationally, including both
+graduate students and senior researchers, come together to participate.
+Workshops help us formulate, test, and revise the benchmark problems and
+allow a valuable venue for other feedback to be provided, such as
```

Comment:
Not a fan of "allow a valuable venue," it feels draconian. Maybe "provide a collaborative atmosphere for brainstorming and constructive criticism," or something along those lines?